.eslintrc.json

```diff
@@ -26,22 +28,21 @@
     "vite*.ts"
   ],
   "plugins": [
+    // put these in the same order as they appear in devDeps for my own sanity :)
```

Comment:
💭 Doesn't the ordering here matter, or is that only the `extends` session where it matters?

Reply:
From what i saw i don't think so, as plugins is just what is going to be used by other configs. the order of the included configs tho does matter

Reply:
Combed through [Configuring Plugins](https://eslint.org/docs/latest/user-guide/configuring/plugins/#configuring-plugins) in the ESLint docs. Don't see anything about having to specify order.